index.js

```diff
@@ -5,7 +5,7 @@ var mold = require('mold-source-map')
   , fs = require('fs');
 
 function separate(src, file, root, base, url) {
-  src.sourceRoot(root || '');
+  src.sourceRoot(root || (src.sourcemap.getProperty('sourceRoot') || ''));
   if (base) {
```

Comment:
Could this not just be written as `src.sourceRoot(root || src.sourcemap.getProperty('sourceRoot') || '');` ? Or am I missing a logic issue here?